pywwt/nbextension/static/interactive_figure/interactive_figure.js

```diff
@@ -172,7 +172,7 @@ function loadImageLayer(layerInfo) {
                                stretchInfo['vmin'],
                                stretchInfo['vmax']);
         wwtLayer.set_colorMapperName(stretchInfo['cmap']);
-        layer.getFitsImage().transparentBlack = false;
+        layer.get_imageSet().get_fitsProperties().transparentBlack = false;
```

Comment:
This kind of thing is why I'm always so concerned with API stability :-) In order to be able to make a new release of pywwt that won't break until the engine update is released, I think we need to simultaneously support both versions of the API. I think this would be the way to do it?

```js
// old transparentBlack API, @wwtelescope/engine <= 7.10. With
// newer engines, this is a harmless no-op.
layer.getFitsImage().transparentBlack = false;

// new transparentBlack API
var fp = layer.get_imageSet()['get_fitsProperties'];
if (typeof fp !== 'undefined') {
    fp.transparentBlack = false;
}
```